test/Microsoft.Restier.Core.Tests/Api.Tests.cs

```diff
@@ -48,7 +48,7 @@ private class TestModelMapper : IModelMapper
                 string namespaceName, string name,
                 out Type relevantType)
             {
-                relevantType = typeof(DateTime);
+                relevantType = typeof (DateTime);
```

Comment:
we don't want extra space between `typeof`/type casting and the left parenthesis. this is Resharper's default convention but not our coding style.